hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/AnalyzeStatementTest.java

```diff
@@ -42,16 +44,54 @@ public static void beforeClass() {
     }
 
     @Test
-    public void test_select() {
+    public void test_options() {
         createMapping("test", Long.class, String.class);
         assertFalse(assertQueryPlan("SELECT * FROM test").isAnalyzed());
         assertTrue(assertQueryPlan("ANALYZE SELECT * FROM test").isAnalyzed());
+        // Default options
         SelectPlan plan = assertQueryPlan(
-                "ANALYZE WITH OPTIONS('opt1'='opt1val', 'opt2'='opt2val') SELECT * FROM test");
+                "ANALYZE WITH OPTIONS("
+                        + "'processingGuarantee'='exactlyOnce', "
+                        + "'snapshotIntervalMillis'='121', "
+                        + "'initialSnapshotName'='pressF', "
+                        + "'maxProcessorAccumulatedRecords'='100'"
+                        + ") SELECT * FROM test");
+        assertTrue(plan.isAnalyzed());
+
+        assertFalse(plan.analyzeJobConfig().isSplitBrainProtectionEnabled());
+        assertFalse(plan.analyzeJobConfig().isAutoScaling());
+        assertFalse(plan.analyzeJobConfig().isSuspendOnFailure());
+
+        assertTrue(plan.analyzeJobConfig().isMetricsEnabled());
+        assertTrue(plan.analyzeJobConfig().isStoreMetricsAfterJobCompletion());
+
+        assertEquals(ProcessingGuarantee.EXACTLY_ONCE, plan.analyzeJobConfig().getProcessingGuarantee());
+        assertEquals(121L, plan.analyzeJobConfig().getSnapshotIntervalMillis());
+        assertEquals("pressF", plan.analyzeJobConfig().getInitialSnapshotName());
+        assertEquals(100, plan.analyzeJobConfig().getMaxProcessorAccumulatedRecords());
+
+        // Test non-default options
```

Comment:
better split that to separate test methods

Reply:
I don't see much sense in that.

Reply:
shorter tests are easier to understand, maintain etc

Reply:
ok